### PR TITLE
chore reworking updatecli manifest

### PR DESCRIPTION
**Pull request by smerle33**
upgrading manifest to be compliant with new scm formating

### CODE REVIEW

**Review by dduportal — CHANGES_REQUESTED** [review]
Blocking the PR: as seen together, we have to update the behavior of the `parallelDockerUpdatecli()` shared library (https://github.com/jenkins-infra/pipeline-library/blob/master/vars/parallelDockerUpdatecli.groovy) to allow passing the docker image to the underlying `updatecli()` library, so we can use the same trick as in https://github.com/jenkins-infra/shared-tools/pull/2 to specify a custom Docker image with both Golang AND updatecli within

**Comment by dduportal** [comment]
Depends on https://github.com/jenkins-infra/pipeline-library/pull/306

File: updatecli/updatecli.d/golang.yaml [hunk 1]
```diff
@@ -64,12 +61,5 @@ targets:
...
-      github:
-        user: "{{ .github.user }}"
-        email: "{{ .github.email }}"
-        owner: "{{ .github.owner }}"
-        repository: "{{ .github.repository }}"
-        token: "{{ requiredEnv .github.token }}"
-        username: "{{ .github.username }}"
-        branch: "{{ .github.branch }}"
+    scmID: default
+
```
Comment (lemeurherve) [inline updatecli/updatecli.d/golang.yaml:65]:
```suggestion
    scmID: default
```

Comment (lemeurherve) [reply updatecli/updatecli.d/golang.yaml:65]: (Ending line without an empty one after)

**Review by dduportal — APPROVED** [review]
LGTM (check is failing, updatecli diff 🥚 / 🐔 as usual)

**Pull request merged**